uaclient/tests/test_config.py

```diff
@@ -0,0 +1,172 @@
+import json
+import os
+
+from nose2.tools.params import params
+
+from uaclient.config import UAConfig
+from uaclient.testing.helpers import TestCase
+
+
+# These are in a variable rather than inline to work around
+# https://github.com/nose-devs/nose2/issues/433
+SIMPLE_PARAMS = (('machine_token', 'machine-token', None),
```

Comment:
This is no longer in use.

Reply:
reused in a later commit. thanks for that sample unit test